plugins/daily-knowledge-sync/skills/daily-knowledge-sync/scripts/categorize_knowledge.py

```
@@ -89,21 +89,35 @@ def categorize(self, text: str, tags: list[str] | None = None) -> str:
+            provided_filename: Optional kebab-case English filename (without date/extension)
 
         Returns:
-            str: Filename (e.g., "2026-01-31_fix_import_error.md")
+            str: Filename (e.g., "2026-01-31_fix-import-error.md")
         """
-        # Clean title for filename
-        clean_title = re.sub(r"[^\w\s-]", "", title.lower())
-        clean_title = re.sub(r"[\s_]+", "_", clean_title)
-        clean_title = clean_title[:50]  # Limit length
+        if provided_filename:
+            # 提供されたファイル名を使用（kebab-case、英語）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)